gulpfile.js

```diff
@@ -233,20 +30,7 @@ gulp.task('default', function (done) {
   done();
 });
 
-function printPackageInfo () {
-  gutil.log(
-    gutil.colors.yellow('v' + pkg.version),
-    gutil.colors.green(pkg.name)
-  );
-  gutil.log();
-  gutil.log(gutil.colors.red(' ______  ______  _____       __   ________  ______  ______'));
-  gutil.log(gutil.colors.red('/\\  ___\\/\\  ___\\/\\  __-.    /\\ \\ / /\\  __ \\/\\__  _\\/\\  ___\\'));
-  gutil.log(gutil.colors.blue('\\ \\  __\\\\ \\  __\\\\ \\ \\/\\ \\   \\ \\ \\\'/\\ \\ \\/\\ \\/_/\\ \\/\\ \\  __\\'));
-  gutil.log(gutil.colors.blue(' \\ \\_\\   \\ \\_\\   \\ \\____-    \\ \\__| \\ \\_____\\ \\ \\_\\ \\ \\_____\\'));
-  gutil.log(gutil.colors.white('  \\/_/    \\/_/    \\/____/     \\/_/   \\/_____/  \\/_/  \\/_____/'));
-  gutil.log();
-}
-
+// TODO: delete or move into separate config if put into use
 //function setBranchBaseUrl() {
   //if (process.env.SITE_BASEURL) {
     //gutil.log(gutil.colors.yellow('set-baseurl'), "Found pre-set SITE_BASEURL: " + process.env.SITE_BASEURL);
```

Comment:
Let's move these two functions into `config/gulp/build.js` so we can call them where needed during the build process.